builtin/provisioners/file/resource_provisioner_test.go

```diff
@@ -48,6 +48,21 @@ func TestResourceProvider_Validate_good_content(t *testing.T) {
 	}
 }
 
+func TestResourceProvider_Validate_good_unknown_variable_value(t *testing.T) {
+	c := testConfig(t, map[string]interface{}{
+		"content":     "74D93920-ED26-11E3-AC10-0800200C9A66",
```

Comment:
I assume this is the "unknown variable value". If possible, it'd be preferable to use one of the global variables that contain this, so that in future when this stops being a magic string we'll be able to track down uses of it to update them.

It's available in a few places, though off the top of my head I'm not sure what's the most convenient to access from here. It's a global variable called `UnknownValue` or `UnknownVariableValue` in a few different packages.